CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 15.0.0
+
+This release replaces four rules with more capable ones.
+
+- Removed: `stylelint` less than `16.13.0` from peer dependencies.
+- Removed: `color-no-invalid-hex` rule.
```

Comment:
[suggestion] This is just an idea, but how about specifically showing "capable ones"? E.g.

```md
- Removed: `color-no-invalid-hex` rule (in favor of `declaration-property-value-no-unknown`).
```

Reply:
I've pushed [a commit](https://github.com/stylelint/stylelint-config-recommended/pull/241/commits/1331028893995674e1d55a71bd7d9c6e495c8b4f) to expand the summary. Each removed rule is replaced by a combo of new ones, so map from old-to-new is tricky.

Reply:
Thank you. I'm sure it'll be beneficial! 👍🏼